python/baseline/dy/classify/train.py

```diff
@@ -48,12 +48,11 @@ def _step(self, loader, update, verbose=False):
 
         metrics = cm.get_all_metrics()
         metrics['avg_loss'] = total_loss / float(steps)
-        if verbose:
-            print(cm)
+        verbose_output_classify(verbose, cm)
```

Comment:
This function has a kind of imprecise name, as it implies that some sort of classification is performed when, in fact, its more of a reporting/output behavior.